facet/src/main/java/com/iluwatar/facet/SecurityMethods.java

```diff
@@ -0,0 +1,7 @@
+package com.iluwatar.facet;
+
+/**
+ * the interface for security methods.
+ */
+public interface SecurityMethods {
```

Comment:
If there are Annotations and attributes that have to be checked at runtime using reflection. Is this empty interfaces used to check at compile-time using the type-system in the compiler?

Reply:
actually, I defined some abstract method for this interface, but it will lower the coverage. So, I empty it.